torch_xla/stablehlo.py

```diff
@@ -233,6 +233,22 @@ def call_function(self, target, args: Tuple, kwargs: Dict) -> Any:
     new_kwargs = dict(kwargs)
     if 'device' in kwargs:
       new_kwargs['device'] = self._device
+    # Note: Use `_disable_current_modes` to alwasys create constant tensor.
+    # Under `fake_tensor_mode` a fake tensor will be created. This is not a
+    # use case for XLAExportInterpreter right now, adding to be future-proof.
+    with torch.utils._python_dispatch._disable_current_modes():
+      # If the op spec expects a `Tensor` input, we wrap the python primitive
+      # type to a torch.tensor. The dtype for float respects
+      # `torch.get_default_dtype`. Without this wrapping, the python float
+      # will be wrapped before it enters dispatcher, and it doesn't respect
+      # the global default dtype.
+      args_and_specs = tuple(zip(args, target._schema.arguments))
+      args = tuple(
+          map(
+              lambda arg_spec: torch.tensor(arg_spec[0])
+              if isinstance(arg_spec[0], (float, int, bool)) and type(arg_spec[
```

Comment:
why boolean is also included here? I understand float and int has 64 bit dtype issue but I thought there is only one boolean dtype

Reply:
Makes sense, let me remove bool